arches_lingo/settings.py

```diff
@@ -110,7 +110,9 @@
         "ENGINE": "django.contrib.gis.db.backends.postgis",
         "HOST": "localhost",
         "NAME": "arches_lingo",
-        "OPTIONS": {},
+        "OPTIONS": {
+            "options": "-c cursor_tuple_fraction=1",
```

Comment:
just curious, what's up with this?

Reply:
It's anticipating https://github.com/archesproject/arches/issues/11382.

I can go ahead and open a PR against the 8.0 project template at this point.

Reply:
Interesting!

Reply:
Yeah, it's good for lingo because when I'm using .iterator() to visit all the broader concept tiles, we know for a fact we need to exhaust all of them to build the correct tree, we won't quit early sometimes.